crates/next-dev/src/lib.rs

```diff
@@ -331,13 +331,19 @@ async fn source(
     .into();
     let static_source =
         StaticAssetsContentSourceVc::new(String::new(), project_path.join("public")).into();
+    let next_static_source = StaticAssetsContentSourceVc::new(
+        "_next/static/".to_owned(),
```

Comment:
Aren't `_next/static` paths already created when referencing an asset? Eg, `import foo from 'path/to/image.jpeg'` would give me a static asset path.

Reply:
I think we can use `SourceAssetVc` to do that.

Reply:
Sorry, it's `StaticAssetVc`